_data/package_index.yml

```
@@ -727,3 +727,11 @@
   categories: examples numerical
   tags: ode pde integral stochastic quadrature plotting
   version: none
+
+- name: Fortran Equation Parser
+  github: FluidNumerics/feq-parse
+  description: An equation parser class for interpreting and evaluating equations provided as strings.
+  categories: strings data-types
```

```suggestion
  categories: strings io
```

Since this is a singleton class (I think), I don't think it's suited to the `data types` category.
The `io` category encompasses parsing, so this is more suitable.

---

I agree. Thanks for the suggestion

---

Hi @schoonovernumerics, just checking have you committed/pushed the above change?

---

@schoonovernumerics, please commit the change if you agree with it (click on the button that says "accept the change" or something like that). Then we can merge this.